api/api.proto

```diff
@@ -290,8 +283,6 @@ message VolumeSpec {
   bool dedupe = 8;
   // SnapshotInterval in minutes, set to 0 to disable snapshots
   uint32 snapshot_interval = 9;
-  // VolumeLabels configuration labels
-  map<string, string> volume_labels = 10;
```

Comment:
We cannot remove the VolumeLabels right away from the Spec. The kubernetes native driver uses that to pass PVC  annotations. We will need to first migrate them to locator.VolumeLabels. I think for now we can mark this field as deprecated and remove its usage.

Reply:
I added it back and labeled as deprecated